docs/docs/theme-ui.md

```diff
@@ -29,8 +29,59 @@ module.exports = {
 Theme UI uses a `theme` configuration object to provide color, typography, layout, and other shared stylistic values through [React context][].
 This allows components within your site to add styles based on a predefined set of values.
 
-The Theme UI plugin uses the [component shadowing API][] to add the theme object context to your site.
-Create a `src/gatsby-plugin-theme-ui` directory in your project, and add an `index.js` file to export a theme object.
+There are two ways to add a theme to your site: using configuration options in `gatsby-plugin-theme-ui`, or local shadowing.
+
+### `gatsby-plugin-theme-ui` configuration
+
+`gatsby-plugin-theme-ui` allows for optional configuration. One of the options you can pass in is `preset`. Preset is a base theme object you'd like your site to use.
+
+The `preset` can be a JSON object or a string package name that the plugin will require for you.
+
+If you're passing in a string package name you need to make sure that the dependency is installed.
+
+```shell
+npm install @theme-ui/preset-funk
+```
+
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-plugin-theme-ui`,
+      options: {
+        preset: "@theme-ui/preset-funk",
+      },
+    },
+  ],
+}
+```
+
+Alternatively, you could do the same thing and require the package yourself. This is necessary if you're referencing a local package that isn't available publicly.
+
+```shell
+npm install @theme-ui/preset-funk
```

Comment:
In this scenario, we've given the example of a local package that isn't available publicly. Could we carry that through to the code snippet (could be a made up name) rather than using the same package as in the previous example? I think that might help avoid some confusion.